cli-plugins/manager/metadata.go

```diff
@@ -24,5 +24,6 @@ type Metadata struct {
 	URL string `json:",omitempty"`
 	// Experimental specifies whether the plugin is experimental.
 	// Experimental plugins are not displayed on non-experimental CLIs.
+	// TODO decide if we want to annotate plugins that are experimental
```

Comment:
/cc @chris-crone @justincormack

Reply:
I dont think we want to annotate them.

Reply:
Removed the TODO, and marked the field deprecated